Tests/js/views/ez-usermenuview.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+<html>
+<head>
+<title>eZ User Menu view tests</title>
+</head>
+<body>
+<div class="container"></div>
+
+<script type="text/x-handlebars-template" id="usermenuitemview-ez-template">
+    <a class="ez-user-menu-item">{{ title }}</a>
+</script>
+
+<script type="text/x-handlebars-template" id="usermenuitemfireeventview-ez-template">
+    <a class="ez-user-menu-item">{{ title }}</a>
+</script>
+
+<script type="text/javascript" src="../../../Resources/public/vendors/yui3/build/yui/yui.js"></script>
+<script type="text/javascript" src="./assets/ez-usermenuview-tests.js"></script>
+<script>
+    var filter = (window.location.search.match(/[?&]filter=([^&]+)/) || [])[1] || 'raw',
+        loaderFilter;
+
+    if (filter == 'coverage'){
+        loaderFilter = {
+            searchExp : "/Resources/public/js/",
+            replaceStr: "/Tests/instrument/Resources/public/js/"
+        };
+    } else {
+        loaderFilter = filter;
+    }
+
+    YUI({
+        coverage: ['ez-usermenuview'],
+        filter: loaderFilter,
+        modules: {
+            "ez-usermenuview": {
+                requires: ['event-tap', 'event-outside', 'template-micro', 'ez-usermenuitemview', 'ez-usermenuitemfireeventview'],
+                fullpath: "../../../Resources/public/js/views/ez-usermenuview.js"
+            },
+            "ez-usermenuitemview": {
+                requires: ['ez-templatebasedview'],
+                fullpath: "../../../Resources/public/js/views/ez-usermenuitemview.js"
+            },
+            "ez-usermenuitemfireeventview": {
```

Comment:
same remark as in https://github.com/ezsystems/PlatformUIBundle/pull/513/files#r53777165
This one should not be loaded and the logout item should be replaced in the test as it's not the component being tested.

for `ez-usermenuitemview`, it depends how you'll test ez-usermenuview. If you inject some mocks as its `items` then it should be removed as well but if you choose to inject *real* item views, then it is needed and should be used.